google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -154,6 +167,7 @@ private StreamingSubscriberConnection(Builder builder) {
 
     subscriberStub = builder.subscriberStub;
     channelAffinity = builder.channelAffinity;
+    protocolVersion = builder.protocolVersion;
```

Comment:
`protocolVersion` is only set when using the `setProtocolVersion` method on the `StreamingSubscriberConnection.Builder`, which can only be called from the `Subscriber` class. Assuming that this is something that we want customers to opt into, we should provide this as an option on the `Subscriber` class that gets propagated to the `StreamingSubscriberConnection` (and defaults to 0 if not set).